.editorconfig

```diff
@@ -14,27 +14,25 @@
 # limitations under the License.
 #
 
-root=true
+root = true
 
 [*]
-charset=utf-8
-end_of_line=lf
-insert_final_newline=true
-trim_trailing_whitespace=true
-indent_style=space
-indent_size=4
-continuation_indent_size=8
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
 
-[{*.yml,*.yaml}]
-indent_style=space
-indent_size=2
+[*.bat]
+end_of_line = crlf
+trim_trailing_whitespace = false
 
-[*.gradle]
-indent_style=space
```

Comment:
Not sure - why removing this here?

Reply:
It's already in the default `[*]` block.